pkg/tsdb/prometheus/framing_test.go

```diff
@@ -151,37 +120,23 @@ func runQuery(response []byte, query PrometheusQuery) (*backend.QueryDataRespons
 
 // we run the mocked query, and extract the DataResponse.
 // we assume and verify that there is exactly one DataResponse returned.
-func testScenario(name string) error {
+func testScenario(t *testing.T, name string) {
```

Comment:
Changed this back to taking `t` because I think it's clearer what's going on if we treat this like the actual test